src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  total: string;
+  perPage: string;
+  currentPage: string;
+  onPageChange: (page: string) => void;
+};
+
+export const Pagination: React.FC<Props> = (
+  {
+    total,
+    perPage,
+    currentPage,
+    onPageChange,
+  },
+) => {
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(+total / +perPage); i += 1) {
+    pageNumbers.push(i);
+  }
+
+  return (
+    <ul className="pagination">
+      <li className={classNames('page-item', {
+        disabled: +currentPage === pageNumbers[0],
+      })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={+currentPage === pageNumbers[0] ? 'true' : 'false'}
```

Comment:
![image](https://user-images.githubusercontent.com/122989585/228640756-adcb5a6d-19b2-4854-b283-2d1fde8161a0.png)
```suggestion
          aria-disabled={(+currentPage === pageNumbers[0]).toString()}
```